plugins/rsyslog.yaml

```diff
@@ -47,6 +47,46 @@ parameters:
     default: ""
     advanced_config: true
     hidden: true
+  - name: tls_enable
+    label: "Enable TLS"
+    description: Should TLS be used?
+    type: boolean
+    default: false
+    advanced_config: true
+    relevant_if:
+      connection_type:
+        equals: "tcp"
+  - name: tls_certificate
+    label: "TLS Certificate"
+    description: Path to the certificate or certificate chain to use for TLS
+    type: string
+    default: ""
+    advanced_config: true
+    relevant_if:
+      tls_enable:
+        equals: true
+  - name: tls_private_key
+    label: "TLS Private Key"
+    description: Path to the certificates private key to use for TLS
+    type: string
+    default: ""
+    advanced_config: true
+    relevant_if:
+      tls_enable:
+        equals: true
+  - name: tls_min_version
+    label: "TLS Minimum Version"
+    description: What minimum version of TLS to use
```

Comment:
```suggestion
    description: Minimum version of TLS to use
```

Reply:
I think we can expand on this, and say it is strictly a minimum, the client will negotiate with the server for a higher version.